web/src/components/storage/ProposalSpacePolicyField.jsx

```diff
@@ -48,22 +48,48 @@ const SPACE_POLICIES = [
   {
     id: "delete",
     label: N_("Delete current content"),
-    description: N_("All partitions will be removed and any data in the disks will be lost.")
+    description: N_("All partitions will be removed and any data in the disks will be lost."),
+    summaryLabels: [
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space deleting all content[...]"
+      N_("deleting all content of the installation device"),
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space deleting all content[...]"
+      N_("deleting all content of the %d selected disks")
+    ]
   },
   {
     id: "resize",
     label: N_("Shrink existing partitions"),
-    description: N_("The data is kept, but the current partitions will be resized as needed.")
+    description: N_("The data is kept, but the current partitions will be resized as needed."),
+    summaryLabels: [
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space shrinking partitions[...]"
+      N_("shrinking partitions of the installation device"),
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space shrinking partitions[...]"
+      N_("shrinking partitions of the %d selected disks")
+    ]
   },
   {
     id: "keep",
     label: N_("Use available space"),
-    description: N_("The data is kept. Only the space not assigned to any partition will be used.")
+    description: N_("The data is kept. Only the space not assigned to any partition will be used."),
+    summaryLabels: [
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space without modifying any partition".
+      N_("without modifying any partition")
+    ]
   },
   {
     id: "custom",
     label: N_("Custom"),
-    description: N_("Select what to do with each partition.")
+    description: N_("Select what to do with each partition."),
+    summaryLabels: [
+      // TRANSLATORS: This is presented next to the label "Find space", so the whole sentence
+      // would read as "Find space performing a custom set of actions".
+      N_("performing a custom set of actions")
```

Comment:
NP: I have asked about the need of using `N_` and `_` here instead of directly `_`. See https://github.com/openSUSE/agama/pull/726#discussion_r1524383469

Asking because I've used directly `_(` in other places and, unless I'm wrong, it works.

Reply:
According to the @lslezak answer, you do not need '`N_` here, unless we decide to start wrapping `Object.freeze` to these objects.